### PR TITLE
[TH2-1443] Migrate from Gauge to Histogram

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,3 +146,7 @@ NOTES:
 
 * in order for the metrics to be exported, you also will need to create an instance of CommonFactory
 * common JVM metrics will also be exported alongside common service metrics
+
+## Release notes
+
+* (2.9.2) - Metrics related to time measurement of an incoming message handling (Raw / Parsed / Event) migrated to the Prometheus [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram)
```
Comment (jean-michel-perli-sonsin) [inline README.md:152]: (2.9.2) - Metrics related to time measurement of an incoming message handling (Raw / Parsed / Event) migrated to **Prometheus**

File: src/main/kotlin/com/exactpro/th2/common/metrics/CommonMetrics.kt [hunk 2]
```diff
@@ -21,6 +21,12 @@ import io.prometheus.client.Gauge
 import java.util.concurrent.CopyOnWriteArrayList
 import java.util.concurrent.atomic.AtomicBoolean
 
+/**
+ * This is a set of default baskets in millisecond for the Prometheus histogram metric.
```
Comment (jean-michel-perli-sonsin) [inline src/main/kotlin/com/exactpro/th2/common/metrics/CommonMetrics.kt:25]: This is a set of default baskets in millisecond for **Prometheus** histogram metric.

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
Hello, I had reviewed the latest changes and left two comments that needed to be changed. The rest of the text itself is correct and looks good/easy to understand.

File: src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/custom/RabbitCustomQueue.kt [hunk 3]
```diff
@@ -98,7 +97,7 @@ class RabbitCustomQueue<T : Any>(
 
         override fun getContentCounter(): Counter = dataCounter
 
-        override fun getProcessingTimer(): Gauge = timer
+        override fun getProcessingTimer(): Histogram? = timer
```
Comment (ivandruzhinin) [inline src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/custom/RabbitCustomQueue.kt:100]:
```suggestion
        override fun getProcessingTimer(): Histogram = timer
```

**Review by jean-michel-perli-sonsin — APPROVED** [review]

**Review by ivandruzhinin — APPROVED** [review]

**Pull request merged**